frontend/util.lua

```diff
@@ -570,6 +570,14 @@ function util.splitToChars(text)
     return tab
 end
 
+function util.lowercaseString(str)
+    if str then
+        require("libs/libkoreader-cre")
+        return cre.lowercaseString(str)
```

Comment:
I could have accepted this as a quick hack in the bit of code involved in FileManager :) but not there in util.lua, which should be and stay rather barebone and requires minimal external stuff.